vms/administrator/tests/test_formFields.py

```diff
@@ -52,21 +47,29 @@ def tearDownClass(cls):
     def check_event_form_values(self, event):
         settings = self.settings
         self.assertEqual(settings.get_event_name_value(), event[0])
+        self.driver.implicitly_wait(10)
         self.assertEqual(settings.get_event_start_date_value(), event[1])
+        self.driver.implicitly_wait(10)
         self.assertEqual(settings.get_event_end_date_value(), event[2])
 
     def check_job_form_values(self, job):
         settings = self.settings
         self.assertEqual(settings.get_job_name_value(), job[1])
+        self.driver.implicitly_wait(10)
         self.assertEqual(settings.get_job_description_value(), job[2])
+        self.driver.implicitly_wait(10)
         self.assertEqual(settings.get_job_start_date_value(), job[3])
+        self.driver.implicitly_wait(10)
         self.assertEqual(settings.get_job_end_date_value(), job[4])
 
     def check_shift_form_values(self, shift):
         settings = self.settings
         self.assertEqual(settings.get_shift_date_value(), shift[0])
+        self.driver.implicitly_wait(10)
         self.assertEqual(settings.get_shift_start_time_value(), shift[1])
+        self.driver.implicitly_wait(10)
         self.assertEqual(settings.get_shift_end_time_value(), shift[2])
+        self.driver.implicitly_wait(10)
         self.assertEqual(settings.get_shift_max_volunteers(), shift[3])
```

Comment:
remove all sleep times please. if you want to have a sleep time - that too > 2 probably there is something wrong.

Reply:
Nevermind. Made some tweaks in my changes and it worked on my test branch.
Waiting for build on this PR.